recipes/clasv/meta.yaml

```diff
@@ -0,0 +1,149 @@
+{% set name = "CLASV" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/JoiRichi/CLASV/archive/refs/tags/v1.0.3.tar.gz
+  sha256: 2d63625311625fd814eb9293721784deeb4550de0464e5030ae927e4f3f87731
+
+build:
+  number: 0
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin="x.x") }}
+  script:
+    - "{{ PYTHON }} -m pip install ."
+    - "{{ PYTHON }} -c 'from CLASV.install_nextclade import install_nextclade; install_nextclade()'"
+
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+    - biopython
+    - pandas
+    - requests
+    - nextclade
+    - plotly
+    - joblib
+    - snakemake
+    - scikit-learn
+
+  run:
+    - python
+    - nextclade
+    - appdirs ==1.4.4
```

Comment:
all those very strict pins should be removed.

A conda package is __not__ a conda environment. Its purpose is to be easy to mix and match with other packages.